agent/globals/globals.go

```diff
@@ -133,5 +134,6 @@ var (
 		AWSAccessKeyId,
 		AWSSecretAccessKey,
 		AWSSessionToken,
+		HostSecret,
```

Comment:
Since the host secret isn't available as an expansion, does it need to be added here?

Reply:
It's not available to the user / task as an expansion but it is added to the expansions within the agent [here](https://github.com/evergreen-ci/evergreen/pull/7714/files#diff-a88b3f432cbdc9d722306e4d87776ee5557beb734f4c4151b5be9a61254cd483R118) so that it can later be [redacted](https://gist.github.com/hadjri/ce8ce76457b741ec724c5339c053a32e#file-redacting_sender-go-L38) from the logs, so I think it is alright / makes sense to keep here.